src/main/resources/migration/changelog-v5.5.0.xml

```diff
@@ -36,4 +36,21 @@
             <column name="PURL"/>
         </createIndex>
     </changeSet>
+
+    <changeSet id="v5.5.0-3" author="sahibamittal">
+        <dropColumn tableName="VULNERABILITY">
+            <column name="EPSSSCORE" type="numeric"/>
+            <column name="EPSSPERCENTILE" type="numeric"/>
+        </dropColumn>
+        <createTable tableName="EPSS">
+            <column autoIncrement="true" name="ID" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" primaryKeyName="EPSS_CVE_PK"/>
+            </column>
+            <column name="CVE" type="TEXT">
```

Comment:
Consider adding a unique index on the `CVE` column, as this is what we'll be always querying on. We also don't want duplicate records for the same CVE.